app/src/main/java/org/catroid/catrobat/newui/ui/fragment/BaseRecyclerListFragment.java

```diff
@@ -141,6 +149,13 @@ private void removeItems(List<T> items) {
 
     protected abstract void cleanupItem(T item) throws Exception;
 
+    private void showRenameDialog()
+    {
+        RenameItemDialog dialog = RenameItemDialog.newInstance(R.string.dialog_rename_item, R.string.rename_item, R.string.rename, R.string.cancel, false);
+        dialog.setRenameItemInterface(this);
+        dialog.show(getFragmentManager(), TAG);
```

Comment:
check if getFragmentManager() is correct. probably should be getSupportFragmentManager ...